include/groonga/groonga.h

```diff
@@ -1520,6 +1520,26 @@ grn_obj_path(grn_ctx *ctx, grn_obj *obj);
 GRN_API int
 grn_obj_name(grn_ctx *ctx, grn_obj *obj, char *namebuf, int buf_size);
 
+/**
+ * \brief Return the name of a column by storing it in `namebuf`.
```

Comment:
↑の`grn_obj_name()`では"an object's name`にしているのでこっちも合わせますか。

```suggestion
 * \brief Return a column's name by storing it in `namebuf`.
```

Reply:
fix: [e684a90](https://github.com/groonga/groonga/pull/2084/commits/e684a90f2f95c02c77826dad9c766cd26c11c277)